klips-api/src/converter.ts

```diff
@@ -134,17 +134,28 @@ const createGeoTiffPublicationJob = (requestBody: any,
       },
       {
         id: 3,
-        type: 'geotiff-optimizer',
+        type: 'dataset-rotation',
         inputs: [
           {
             outputOfId: 2,
             outputIndex: 0
           },
```

Comment:
The above lines specify the input GeoTiff, e.g. `/opt/staging/langenfeld_20230628T1200Z.tif`, right?

Reply:
yep, that's correct.